inform7/Internal/Inter/Architecture32Kit/Sections/Input Output.i6t

```diff
@@ -256,6 +266,10 @@ to document all of that.
         glk_put_char_stream(gg_commandstr, 10); ! newline
     }
   .KCPContinue;
+	@binarysearch res WORDSIZE FunctionKeyCodes WORDSIZE FunctionKeyCodesCount 0 4 ix;
```

Comment:
I should first just check if `res` is negative, no need to even do the binary search if not.